homeassistant/components/sensor/linky.py

```diff
@@ -75,17 +78,22 @@ def unit_of_measurement(self):
     @Throttle(SCAN_INTERVAL)
     def update(self):
         """Fetch new state data for the sensor."""
-        from pylinky.client import PyLinkyError
+        from pylinky.client import LinkyClient, PyLinkyError
+
+        client = LinkyClient(self._username, self._password, None,
```

Comment:
Why do we need to create a new client on every update?

Reply:
First solution use one client, created in setup_platform, but it's seems to not work (Login error).
If we create a new client on every update, it works.

Reply:
It would be good to know why?

Reply:
I agree with you. 
My solution was a hotfix what work for me and closing error login issue, it's why i share it. In wait I've found a better solution, do you want I cancel my PR ?

Reply:
We shouldn't necessarily close this. Let's hear what other reviewers think. If this PR fixes the problem, that's a first step.

Reply:
After some tests, I agree with @hobbe : pyLinky library session cannot be re-used.
So, for now, we need to re-create an client at each update.

Reply:
I suggest that you open an issue for pyLinky to get it fixed.